### PR TITLE
Database diagram created

### PR DESCRIPTION
**Pull request by massazoum**
In this pull request ,

- We Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram below:
image
-We added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  FOREIGN KEY (invoice_id) REFERENCES invoices (id),
+  FOREIGN KEY (treament_id) REFERENCES treaments (id)
+);
+CREATE TABLE treaments (
+  id SERIAL PRIMARY KEY,
+  type INT,
+  name VARCHAR(225),
+  medical_history_id INT,
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id)
+);
```
Comment (christianbiring1) [inline schema_based_on_diagram.sql]: - [ ] Kindly make sure all FOREIGN KEYS have indexes created as specified in the projects requirements.
Hint:
```sql
CREATE INDEX ON medical_histories (patient_id);
```

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @massazoum @harshi0102 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Great work on implementing the database 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (danmainah) [inline schema_based_on_diagram.sql:41]: - [ ] [optional] @massazoum and @harshi0102 great work implementing the changes requested, however, you forgot to create all "join table" for the many-to-many relationships, between `medical_histories` and `treatments` , also you dont have a foreign key on the treatments table the relationship is suppose to be covered using the join table, kindly consider implementing the join table and add indexes on its foreign keys
```
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

**Review by danmainah — APPROVED** [review]
Hi @massazoum and @harshi0102 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good PR and Description, with information on what was a achieved in the pull request

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**